src/js/rx/components/Disclaimer.jsx

```diff
@@ -1,26 +1,44 @@
 import React from 'react';
+import classNames from 'classnames';
+
 
 class Disclaimer extends React.Component {
   render() {
+    const openClass = classNames(
+      { 'rx-disclaimer--open': this.props.isOpen }
```

Comment:
Consolidate class names:

    const disclaimerClass = classNames(
      'rx-disclaimer',
      'cf',
      { 'rx-disclaimer--open': this.props.isOpen }
    );